### DISCUSSION TITLE
Leicht Off Topic: Externen Antennenanschluss mittels SMA Verlängerungskabel verlängern möglich?

### DISCUSSION THREAD

**Discussion by timosust** [jomjol/AI-on-the-edge-device#1618] [Q&A]
Moin,

ich habe leider bisher im Netz nichts dazu finden können. Vielleicht weis das zufällig jemand hier.

Wenn ich eine externe Antenne bei der esp32-cam verwende, habe ich ja diese kurze Kabelpeitsche von Ipex auf SMA. Und dann wird daran die Antenne angeschraubt.

Wäre es auch möglich, diese Leitung mit einem entsprechenden SMA Antennenverlängerungskabel um ein paar Meter zu verlängern, um die Antenne näher am Access Point platzieren zu können?

Oder gibt das dann Probleme wegen Resonanz, zu hohem Leitungswiderstand, etc?

Thx

**Comment by caco3** [comment]
Grundsätzlich sollte ein Verlängerunsgkabel kein Problem sein. Solche Kabel haben eine definierte Impedanz (z.b. 50 Ohm), dadurch hat die Länge nahezu keinen Einfluss.

Ich meine, der Anschluss auf Geräteseite heist I-PEX und auf Antennenseite brauchst vermutlich ein SMA:
 - https://www.dfrobot.com/product-2382.html
 - https://www.reichelt.com/ch/de/index.html?ACTION=446&LA=3&nbc=1&q=i-pex%20zu%20sma


Ev. hilft auch das, wenn Du selber basteln willst: https://marksbench.com/electronics/esp32-cam-antenna-workaround/

**Comment by stefan123t** [comment]
Keine Ahnung ob bei den ESP32 CAM Modulen auch ein 0 Ohm Widerstand umgelötet werden muß wie bei einigen ESP8266 / ESP32 Modulen ohne Kamera, damit der externe Antennenanschluß überhaupt aktiv wird ?

**Reply by ghost** [reply]
Ja muss. Ist etwas fummelig das kleine Ding, aber notwendig, wenn die Antenne auch etwas bringen soll.

Bei mir hatte ich bei zwei Geräten das Problem im Keller. Eines im Zählerschrank. Dort habe ich die Antenne nach draußen gelegt und komme jetzt auf RSSI: -72dBm.

**Reply by timosust** [reply]
Jupp, muss definitiv. Wollte das von Anfang an machen und hatte mir gleich die erste Cam gehimmelt.

Jetzt habe ich erstmal provisorisch nen billigen Allnet Repeater in der Waschküche hängen, der scheint allerdings mit der FritzBox immer mal wieder Probleme zu machen. Interessanterweise habe ich dessen Netz gelegentlich sogar bis hier oben im 2.OG, während das Mesh von FritzRepeater trotz 5m Luftlinie nur an einer kleinen Ecke im Waschkeller empfangbar scheint.

Ziel wäre es den Allnet Repeater weg zu bekommen. Andererseits will ich den Gaszähler, der bisher mit einem schnöden Reed Fenstersensor ausgelesen u in HA verarbeitet wird, noch auf nen D1 Mini umstellen (darauf läuft dann ein Counter für den Reedkontakt, und meldet Absolutwerte per MQTT), damit ich Verbindungsausfälle zu HA kompensieren kann. Muss ich dann sehen, ob das mit dem D1 Mini Pro mit externer Antenne ausreicht.

Die ESP32 Cam scheidet hier aus, weil zu wenig Platz für ausladendes Gehäuse vorhanden.

Hach ja, das Projekt wird immer teurer ;-)
